src/components/ChooseADate.jsx

```diff
@@ -0,0 +1,236 @@
+    // Récupération de l'API et changement des 3 paramètres
+    axios
+      .get(
+        `https://api.nasa.gov/planetary/apod?api_key=X0pTCLU1WDtehvcdyz3Ju71cdobzX2RzjUDLTzk8&date=${year}-${month}-${day}`
+      )
+      .then((response) => response.data)
+      .then((data) => {
+        console.log(data);
+        this.setState({
+          photo: data,
+        });
+      });
+  }
+
+  //Au changement du mois, la nouvelle valeur de month devient monthValue
+  handleChangeMonth = (e) => {
+    const monthValue = e.target.value;
+    console.log(monthValue);
+    this.setState({ month: monthValue });
+  };
+
+  handleChangeDay = (e) => {
+    const dayValue = e.target.value;
+    console.log(dayValue);
+    this.setState({ day: dayValue });
+  };
+
+  handleChangeYear = (e) => {
+    const yearValue = e.target.value;
+    console.log(yearValue);
+    this.setState({ year: yearValue });
+  };
+
+  //Quand on clique sur submit on modifie les paramètres de la fonction getGalerie et on l'active
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.getGalerie(this.state.year, this.state.month, this.state.day);
+  };
+
+  render() {
+    return (
+      <Page>
+        <Textcolor>
+          <BaliseSelection>
+            <form onSubmit={this.handleSubmit}>
+              <label>
+                <Select value={this.state.day} onChange={this.handleChangeDay}>
+                  <option value="day">Day</option>
+                  <option value="1">1</option>
+                  <option value="2">2</option>
+                  <option value="3">3</option>
+                  <option value="4">4</option>
+                  <option value="5">5</option>
+                  <option value="6">6</option>
+                  <option value="7">7</option>
+                  <option value="8">8</option>
+                  <option value="9">9</option>
+                  <option value="10">10</option>
+                  <option value="11">11</option>
+                  <option value="12">12</option>
+                  <option value="13">13</option>
+                  <option value="14">14</option>
+                  <option value="15">15</option>
+                  <option value="16">16</option>
+                  <option value="17">17</option>
+                  <option value="18">18</option>
+                  <option value="19">19</option>
+                  <option value="20">20</option>
+                  <option value="21">21</option>
+                  <option value="22">22</option>
+                  <option value="23">23</option>
+                  <option value="24">24</option>
+                  <option value="25">25</option>
+                  <option value="26">26</option>
+                  <option value="27">27</option>
+                  <option value="28">28</option>
+                  <option value="29">29</option>
+                  <option value="30">30</option>
+                  <option value="31">31</option>
```

Comment:
I heard about a `for` instruction, very useful for this kind of thing... :smirk: